app/views/outreach_events/edit.html.erb

```diff
@@ -2,4 +2,10 @@
  
 <%= render 'form' %>
  
-<%= link_to 'Back', outreach_events_path %>
\ No newline at end of file
+<% if current_user.admin? %>
+  <%= link_to 'Back', admin_outreach_events_path %>
+<% elsif current_user.facilitator? %>
```

Comment:
This shouldn't be necessary if we remove the facilitator's outreach events route and change the `<% elsif current_user.student? %>` to an `<% else %>`